static/app/utils/replays/useReplayData.tsx

```diff
@@ -109,55 +114,40 @@ function useReplayData({eventSlug, orgId}: Options): Result {
         const response = await api.requestPromise(
           `/api/0/projects/${orgId}/${projectId}/events/${eventId}/attachments/${attachment.id}/?download`
         );
-        return JSON.parse(response).events as eventWithTime;
+        return JSON.parse(response) as ReplayAttachment;
       })
     );
-    return attachments.flat();
-  }, [api, eventId, orgId, projectId]);
-
-  const fetchReplayEvents = useCallback(async () => {
-    const replayEventList = await api.requestPromise(
-      `/organizations/${orgId}/eventsv2/`,
-      {
-        query: {
-          statsPeriod: '14d',
-          project: [],
-          environment: [],
-          field: ['timestamp', 'replayId'],
-          sort: 'timestamp',
-          per_page: 50,
-          query: ['transaction:sentry-replay-event', `replayId:${eventId}`].join(' '),
-        },
-      }
-    );
 
-    return Promise.all(
-      replayEventList.data.map(
-        event =>
-          api.requestPromise(
-            `/organizations/${orgId}/events/${generateEventSlug(event)}/`
-          ) as Promise<Event>
-      )
-    );
-  }, [api, eventId, orgId]);
```

Comment:
No longer needed as these were used to aggregate breadcrumbs + performance spans.